### PR TITLE
update challenge instruction js-inputs-and-strings multiple-inputs

### PR DESCRIPTION
**Pull request by FelineHuhn**
Outdated description of js-inputs-and-strings multiple-inputs challenge replaced

related to Ticket [#662](https://github.com/orgs/neuefische/projects/45/views/10?pane=issue&itemId=61129727)

### CODE REVIEW

**Review by F-Kirchhoff — COMMENTED** [review]
Hey Feline,
this wording is much better than the current verison. What do you think about removing the need for commenting out code entirely? We could triple the html inputs and use each for one part respectively. This might improve the students understanding of the different mechanics since they can compare them side by side.

File: sessions/js-inputs-and-strings/multiple-inputs/css/styles.css [hunk 1]
```diff
@@ -7,16 +7,17 @@
 body {
   margin: 0;
   font-family: system-ui;
+  background-color: #ddd;
 }
 
 .form {
   display: grid;
   grid-gap: 20px;
```
Comment (F-Kirchhoff) [inline sessions/js-inputs-and-strings/multiple-inputs/css/styles.css:15]: the more modern property would be to use gap instead
```suggestion
  gap: 20px;
```

File: sessions/js-inputs-and-strings/multiple-inputs_solution/css/styles.css [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Helvetica,
-    Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+  font-family: system-ui;
+  background-color: #ddd;
 }
 
 .form {
   display: grid;
   grid-gap: 20px;
```
Comment (F-Kirchhoff) [inline sessions/js-inputs-and-strings/multiple-inputs_solution/css/styles.css:15]: same here
```suggestion
  gap: 20px;
```

File: sessions/js-inputs-and-strings/multiple-inputs_solution/index.html [hunk 3]
```diff
@@ -6,19 +6,69 @@
...
+      />
+      <button
+        class="form__button"
+        type="button"
+        data-js="copy-uppercase-button"
+      >
+        Click to copy the first input value into the second input field in
+        uppercase
+      </button>
+    </form>
```
Comment (F-Kirchhoff) [inline sessions/js-inputs-and-strings/multiple-inputs_solution/index.html:54]: what do you think about giving each form an h2 for the challenge part? 
```suggestion
    <form class="form">
      <h2>Part 2: Copy and UpperCase</h2>
      <label for="first-copy-uppercase-input">First Input</label>
      <input
        class="form__input"
        id="first-copy-uppercase-input"
        data-js="first-copy-uppercase-input"
      />
      <label for="second-copy-uppercase-input">Second Input</label>
      <input
        id="second-copy-uppercase-input"
        class="form__input"
        data-js="second-copy-uppercase-input"
      />
      <button
        class="form__button"
        type="button"
        data-js="copy-uppercase-button"
      >
        Click to copy the first input value into the second input field in
        uppercase
      </button>
    </form>
```

**Review by F-Kirchhoff — COMMENTED** [review]
Hey Feline,
great implementation and nice variable names. What do you think about giving each form a title on the website?

**Comment by FelineHuhn** [comment]
Great idea for even better clarity @F-Kirchhoff 👍 I have now also added this.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🚀

**Pull request merged**